website/src/_posts/2019-05-11-beam-summit-europe-2019.md

```diff
@@ -0,0 +1,88 @@
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# The Apache Beam community in 2019
+
+
+2019 has already been a busy time for the Apache Beam. The ASF blog featured [our way of community building](https://blogs.apache.org/comdev/date/20190222) and we've had [more Beam meetups](https://www.meetup.com/San-Francisco-Apache-Beam/events/257482350)  around the world. Apache Beam also received the [Technology of the Year Award](https://www.infoworld.com/article/3336072/infoworlds-2019-technology-of-the-year-award-winners.html) from InfoWorld.
+
+As these events happened, we were building up to the [20th anniversary of the Apache Software Foundation](https://opensource.googleblog.com/2019/03/celebrating-20-years-of-apache.html).  The contributions of the Beam community were a part of Maximilian Michels blog post on the success of the ASF's open source development model:
+
+<blockquote class="twitter-tweet" data-lang="nl"><p lang="en" dir="ltr">Success at Apache: What You Need to Know by Maximilian Michels <a href="https://t.co/XjtVYgPAHX">https://t.co/XjtVYgPAHX</a> <a href="https://twitter.com/hashtag/Apache?src=hash&amp;ref_src=twsrc%5Etfw">#Apache</a> <a href="https://twitter.com/hashtag/Open?src=hash&amp;ref_src=twsrc%5Etfw">#Open</a> <a href="https://twitter.com/hashtag/Innovation?src=hash&amp;ref_src=twsrc%5Etfw">#Innovation</a> <a href="https://twitter.com/hashtag/Community?src=hash&amp;ref_src=twsrc%5Etfw">#Community</a> <a href="https://twitter.com/hashtag/people?src=hash&amp;ref_src=twsrc%5Etfw">#people</a> <a href="https://twitter.com/hashtag/processes?src=hash&amp;ref_src=twsrc%5Etfw">#processes</a> <a href="https://twitter.com/hashtag/JustWorks?src=hash&amp;ref_src=twsrc%5Etfw">#JustWorks</a> <a href="https://twitter.com/stadtlegende?ref_src=twsrc%5Etfw">@stadtlegende</a> <a href="https://t.co/xSibnyWAMe">pic.twitter.com/xSibnyWAMe</a></p>&mdash; Apache - The ASF (@TheASF) <a href="https://twitter.com/TheASF/status/1110364656143601664?ref_src=twsrc%5Etfw">26 maart 2019</a></blockquote>
+
+<script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+In that spirit, let's have an overview of the things that have happened, what the next few months look like, and how we can foster even more community growth.
+  
+
+## Meetups
+We've had a flurry of activity, with several meetups in the planning process and more popping up globally over time. As diversity of contributors is a core ASF value, this geographic spread is exciting for the community. Here's a picture from the latest Apache Beam meetup organized at Lyft in San Francisco:
+
+<img src="{{ "/images/blog/Lyft.jpg" | prepend: site.baseurl }}" alt="Beam Meetup Bay Area" >
```

Comment:
It would be nice to find a way to not add these images to the Beam repository. (I like the way you include the Tweets.)

Reply:
Agreed, this is just how I did it in the past. External hosting is an option, but would also add a dependency. This way, we are sure the blogpost will look nice in the future.